chore: changelog notes (#2790)
vuestorefront/storefront-ui: CHANGELOG.md

# 2.3.0 (2023-06-07)

## Overview

> This release introduces new components, blocks and improvements of existing blocks.

Reviewer: ```suggestion
This release introduces SfBadge alongside new blocks and improvements.
```

## Features

### Blocks Vue.js and React.js

- Filters: Category, Rating, Price and Brand blocks
- Combobox
- Search: with custom button and with custom icon button

> - Gallery: with arrow keys navigation 
> ### Components Vue.js and React.js

Reviewer: ```suggestion
- Gallery: with arrow keys navigation 

### Components Vue.js and React.js
```

> - SfBadge
> ## Improvements

Reviewer: ```suggestion
- SfBadge

## Improvements
```

- docs: possibility to turn off padding in example iframe 

> - chore: CI building process improved 

Reviewer: ```suggestion
- chore: introduced changesets and continous development pipeline
```

- useScrollable: add missing methods, remove prop getter and add containerRef  

> - chore: EsLint speed improvement

Reviewer: ```suggestion
- chore: ESLint speed improvement
```
